matrix_client/room.py

```diff
@@ -29,9 +29,11 @@ def __init__(self, client, room_id):
         self.events = []
         self.event_history_limit = 20
         self.name = None
+        self.canonical_alias = None
         self.aliases = []
         self.topic = None
         self._prev_batch = None
+        self._members = []
```

Comment:
For a future feature, it'd be really nice if this was automatically kept up-to-date by the sync thread in the same way as topic. aliases, name, etc.

Reply:
Done. Or at least i think its the right place.